com.notepadqq.Notepadqq.yaml

```diff
@@ -0,0 +1,43 @@
+app-id: com.notepadqq.Notepadqq
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.11'
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: notepadqq
+rename-icon: notepadqq
+rename-desktop-file: notepadqq.desktop
+rename-appdata-file: notepadqq.appdata.xml
+copy-icon: true
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Filesystem access
+  # for Open Folder
+  - --filesystem=host
+  # Network access
+  # Qt: Session management error: Could not open network socket
+  - --share=network
+  # Dconf access
+  # dconf-WARNING **: failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  # Portal
+  # https://github.com/notepadqq/notepadqq/issues/460#issuecomment-384370580
+  # https://github.com/notepadqq/notepadqq/issues/552#issuecomment-377674942
+  # https://github.com/flatpak/flatpak/issues/1957#issuecomment-412860191
+  #- --env=XDG_CURRENT_DESKTOP=KDE
+  # OpenGL access
+  # libGL error: failed to open drm device: No such file or directory
+  # libGL error: failed to load driver: i965
```

Comment:
I fail to see why these are helpful comments.

Reply:
> I fail to see why these are helpful comments.

This explains why do we need such an option and what will happen if it is disabled.
You constantly ask me about every single option and I can't remember everything.

Reply:
While I appreciate trying to document things sometimes its just too obvious and becomes noise.

DRI access is for OpenGL. AFAIK all Qt applications use OpenGL. This does not need to be documented.

```yaml
# Dconf access
- --filesystem=xdg-run/dconf
```

That is already clear. And like @barthalion was saying you should understand *why* permissions exist. You don't add a permission to silence a warning. The only reason you probably need DConf access is for the Qt GNOMEPlatform module on Wayland.

Reply:
> How am I supposed to know what is obvious to you and what is not? Sometimes you ask me for really obvious things, like why a Gtk+2 graphics tool (that supports GIMP/Inkscape files) needs some level of the --filesystem permissions.

It is pretty simple. The level of knowledge your comments should be targeting is somebody familiar with Flatpak but not familiar with the application.

The reason `--filesystem` is questioned is because its an awful permission that completely throws security out the window and should be avoided whenever possible. As a reviewer I assume that packagers are the opposite, they know the application but not Flatpak.

So given that baseline of how reviewer and maintainer interact here do you understand why the DRI permission is obvious and why network permissions on a text editor we've never used isn't?